src/hooks/useCioBreadcrumb.ts

```diff
@@ -0,0 +1,35 @@
+import { Group } from '@constructor-io/constructorio-client-javascript/lib/types';
+import { useMemo } from 'react';
+
+const generateBreadcrumbs = (groups: Partial<Group>[], filterValue: string) => {
```

Comment:
In my PR with groups, I made a new type `PlpGroup` which is essentially the same thing, but with the fields transformed to camel case to match what were doing with the other fields. I think this is fine for now, I'll note down a story to make it consistent later on.